engine/schema/src/com/cloud/network/dao/NetworkDaoImpl.java

```diff
@@ -377,11 +377,16 @@ protected void addAccountToNetwork(final long networkId, final long accountId, f
     }
 
     @Override
-    public String getNextAvailableMacAddress(final long networkConfigId) {
+    public String getNextAvailableMacAddress(final long networkConfigId, Integer zoneMacIdentifier) {
```

Comment:
I tried writing unit test cases for it. I have difficulty in call static method. I tried with the PowerMockito.mockStatic but still I face NPE in fetch.getNextSequence.